src/main/resources/edu/cornell/kfs/coa/cu-coa-resources.properties

```diff
@@ -1,2 +1,6 @@
 error.document.accountMaintenance.icrTypeCodeCannotBeEmpty=When {0} is {1}, at least one ICR Type Code is required.
 error.document.accountMaintenance.icrSeriesIdentifierCannotBeEmpty=When {0} is {1}, at least one ICR Series Identifier is required.
+
+legacy.account.attachment.note.text.template=Legacy Account Number: %1$s\
```

Comment:
I don't see where the functionals agreed with my proposed note message for these attachments, we should confirm that.

Reply:
I'll check with Cathy when she returns this week.